android/console.c

```diff
@@ -1801,6 +1824,10 @@ static const CommandDefRec  gsm_commands[] =
     "namePresentation range is 0..2\r\n",
     NULL, do_gsm_call, NULL },
 
+    { "enable_hold", "enable/disable the hold feature.",
```

Comment:
How about using `gsm set hold <true/false>` for the command format?
With this format, we can easily extend it to support other feature, if we have to enable/disable some feature in the future. e.g. `gsm set foo <true/false>` ...
What do you think?

I prefer using `gsm set hold <true/false>` for the command format.

What do you think?